docs/bin/plugin_formatter.py

```diff
@@ -58,6 +58,9 @@ def html_escape(text, quote=True):
 from ansible.utils.display import Display
 from ansible.utils._build_helpers import update_file_if_different
 
+sys.path.append('../../test/sanity/validate-modules')
+from module_args import AnsibleModuleImportError, get_argument_spec
```

Comment:
Oh, very nice.

Reply:
IMO it's an ugly hack. I think this functionality should move into Ansible to be used both by test-scripts and doc-generators.

Reply:
*Exactly*